.grunt-config/sass.js

```diff
@@ -79,9 +79,9 @@ const sass = {
 			},
 			{
 				expand: true,
-				cwd: 'assets/dev/scss/frontend',
-				src: 'shapes.scss',
-				dest: 'assets/css',
+				cwd: 'assets/dev/scss/frontend/conditionals',
+				src: '*.scss',
+				dest: 'assets/css/conditionals',
```

Comment:
I have moved this to its own folder, to ensure that we don't overwrite any files inside the `/css` folder.